features/helpers.rb

```diff
@@ -0,0 +1,23 @@
+# In order to avoid repeating operations supposed to take place x times in various tests, I created this module
+# This article helped a lot: http://blog.pivotal.io/labs/labs/cucumber-step-definitions-are-not-methods
+
+module Helpers
+  def signup(user)
+    visit "/users/sign_out"
```

Comment:
Exactly: I wanted to make sure user is not signed when starting the test, but I realize it's completely useless since we have complete control over user state in the tests.